final/problem-def-ana/require-ana.tex

```diff
@@ -4,31 +4,36 @@ \section{Requirements Analysis}
 Defining requirements upfront is important for several reasons: it is a contract
 between the developers and the client, it guides the product development, it
 enables the client to track the progress and finally it allows validation of the
-deliverable. These requirements are listed in this section.
+deliverable. The requirements are listed in this section.
 
 \subsection{Design goals}
 \label{ssec:goals}
 
 The client has expressed some high-level requirements, which are listed in this
 subsection as design goals in order of priority. The design goals serve as an
-important guideline when defining and implement the invidual requirements. As
+important guideline when defining and implementing the invidual requirements. As
 such, they can be considered the bounds within which all requirements must fit.
 
-\paragraph{Language-agnostic} The REPL should work with all languages
-defined with Spoofax. This means that no assumptions can be made about any
-language constructs or syntax.
+\paragraph{Language-agnostic} The REPL should not make any assumptions about its
+host language: it must work with all languages defined with Spoofax. This also
+means that a language designer should not have to do any additional work to get
+a REPL for their language. However, if required, the language designer should be
+able to provide REPL-specific configuration.
 
+% FIXME: what to do with this one? We don't generate a REPL: it is always there.
```

Comment:
This one can either be removed altogether, or changed to a paragraph about the REPL being easily accessible without unnecessary manual labor.